package foo

import "fmt"

func f() error {
	1 + try(fmt.Println("hello"))
}
